pages/index.js

```diff
@@ -1,36 +1,15 @@
-import PropTypes from 'prop-types';
-
-import getStaticData from '../src/services/get-staticData';
-import Spot from '../src/components/Spot/Spot';
-import Animal from '../src/components/Animal/Animal';
-import Organizer from '../src/components/Organizer/Organizer';
-import ElementList from '../src/components/ElementList/ElementList';
+import useStore from '../src/lib/useStore';
 import Map from '../src/components/Map/Map';
+import { useEffect } from 'react';
 
-export function getStaticProps() {
-	const spots = getStaticData('spots');
-	const animals = getStaticData('animals');
-	const organizers = getStaticData('organizers');
+export default function Home() {
+	const fetchData = useStore(state => state.fetchData);
 
-	return { props: { spots: spots, animals: animals, organizers: organizers } };
-}
+	useEffect(() => {
+		fetchData('spots');
+		fetchData('animals');
+		fetchData('organizers');
+	}, [fetchData]);
 
-export default function Home({ spots, animals, organizers }) {
-	return (
-		<>
-			<Spot spot={{ ...spots[1] }} animals={animals} organizers={organizers} />
-			<Animal spots={spots} animal={{ ...animals[0] }} organizers={organizers} />
-			<Organizer organizer={{ ...organizers[2] }} />
-			<ElementList elements={spots} header="Spots" />
-			<ElementList elements={animals} header="Animals" />
-			<ElementList elements={organizers} header="Organizers" />
```

Comment:
If I didn't miss something this components are not used anymore.

- Spot
- Animal
- Organizer
- ElementList

Can the be deleted? Are they in place for further implementations?

Reply:
Yes, they are in place for the upcoming navigation, where one can choose between globe-mode and list-mode - but I can easily delete them for now and implement everything again later, if this is better ?

Reply:
Having Code on place that is not used any more is a bad practice in general.

Since you used this component in the previous iteration and are going to use them in the next iteration I would keep them. The current iteration seems to be a very fluid situation.